src/api/v1/teams/team_controllers.js

```diff
@@ -0,0 +1,123 @@
+import Team from './team_model'
+const User = require('../users/users_model')
+import { InternalServerError, CustomError } from '../../../utils/customError'
+import responseHandler from '../../../utils/responseHandler'
+
+export async function createTeam(req, res, next) {
+    try {
+        const { team_name, mentorId, trackId } = req.body
+        let teamNumber
+        const lastEntry = await Team.findOne().sort({ $natural: -1 }).limit(1)
+        if (lastEntry) {
+            // team names starting with SH- for shoman
+            teamNumber = `SH-${parseInt(team.team_number.slice(3)) + 1}` // generate team number from last entry
+        } else {
+            teamNumber = 'SH-101' // starting with 101 but could be changed
```

Comment:
Maybe name it in full
```suggestion
            teamNumber = 'SHOMAN-101' // starting with 101 but could be changed
```